core/src/components/datetime/test/basic/datetime.e2e.ts

```diff
@@ -323,7 +323,8 @@ configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, config }) => {
  */
 configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
   test.describe(title('datetime: visibility'), () => {
-    test('should reset month/year interface when hiding datetime', async ({ page }) => {
+    // TODO NOW
+    test.skip('should reset month/year interface when hiding datetime', async ({ page }) => {
```

Comment:
Was this intentionally left in?

Reply:
For now, yes. I need to report this to the WebKit team. I skipped the test so we could al least get screenshots updated/the rest of the changes reviewed.

Reply:
Could we add a proper tech debt ticket to remove the skip once the Webkit issue is resolved?

Reply:
Yeah my plan is to add a tech debt ticket once I report this to the WebKit team. I'm still trying to create an isolated test case.

Reply:
added in https://github.com/ionic-team/ionic-framework/pull/29046/commits/1dc4cb5f02dfec935d4fb927318a2d4f12dc0b4c